webassembly-observe-sdk/README.md

```diff
@@ -0,0 +1,55 @@
+# Dylibso WebAssembly Observe SDK
+
+## Overview
+
+This integration provides function traces from WebAssembly (WASM) code running in your application. Gain insight into WebAssembly code performance as well as the following behavior:
+- Function call duration
+- Execution tracing
+- Memory allocation
+
+Since WebAssembly code is executed in a secure and constrained environment, traditional techniques to monitor code do not work. Our specialized observability stack
+allows you to continuously monitor WASM modules at the same level you expect of your other applications.
+
+Datadog customers can use our open source SDKs and Adapters to emit full traces from your WASM programs.
+
+Please see the [`dylibso/observe-sdk`][2] repository to install the Datadog Adapter for your application.
+
+In addition, Dylibso provides automatic instrumentation tooling which can take any existing WASM module and recompile it to include function and memory allocation tracing. For
+more information, contact [support@dylibso.com](mailto:support@dylibso.com) or learn more about [automatic WebAssembly instrumentation][3].
```

Comment:
@estherk15 is this the correct formatting for the email?

Reply:
@emarsha94 Thanks for verifying. Yes, this is correct!